### DISCUSSION TITLE
Focus opened file in editor on `enter` from project_panel

### DISCUSSION THREAD

**Discussion by Kochanac** [zed-industries/zed#17810] [Feature Requests]
Hello. I'm trying to emulate VSCode default behaviour that when focused in project file tree, I could type "enter" and open+focus cursor in the editor of selected file. I've tried the following keymap:

```json
    {
        "context": "ProjectPanel",
        "bindings": {
            "enter": ["project_panel::Open", "project_panel::ToggleFocus"]
        }
    }
```

But, unfortunately, project_panel::ToggleFocus does nothing. I've also tried editor::ToggleFocus

**Comment by educbraga** [comment]
@Kochanac  I could solve this by sending keystrokes:

```
[
  {
    "context": "ProjectPanel && not_editing",
    "bindings": {
      "enter": ["workspace::SendKeystrokes", "space cmd-shift-e"],
      "cmd-enter": ["workspace::SendKeystrokes", "f2"]
    }
  }
  ]
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
